packages/hosted-widget-integration/src/HostedWidgetPaymentComponent.tsx

```diff
@@ -500,7 +503,7 @@ class HostedWidgetPaymentComponent extends Component<
     }
 
     private handleUseNewCard: () => void = async () => {
-        const { deinitializePayment = noop, initializePayment = noop, method } = this.props;
```

Comment:
Remove unnecessary `noop` because `deinitializePayment` is a required prop.
So the code coverage got improved.